scenarioo-server/src/main/java/org/scenarioo/rest/application/ScenariooWebApplication.java

```diff
@@ -60,41 +61,15 @@ public void contextInitialized(final ServletContextEvent servletContextEvent) {
 	private void loadConfiguration(final ServletContextEvent servletContextEvent) {
 		LOGGER.info("  Loading configuration ...");
 
-		final String configurationDirectory = configureConfigurationDirectoryFromServerContext(servletContextEvent);
-		final String configurationFilename = configureConfigurationFilenameFromServerContext(servletContextEvent);
+		final String configurationDirectoryPath = documentationPathLogic.getDocumentationPath(servletContextEvent);
```

Comment:
@mi-we Please log the configured path here - before reading the config from there - in case this is failing, I want to see which path is configured.

Reply:
did it